e2e_playwright/multipage_apps_v2/mpa_v2_basics.py

```diff
@@ -65,14 +66,19 @@ def get_input():
     get_input()
 
 
+def page_14():
+    time.sleep(0.5)
+    st.number_input("mynum", value=1, key="mynum")
```

Comment:
I think it could make sense to test this behavior also within a fragment-decorated function. I have tested it manually and it works, but since fragment and app reruns are slightly different (the one uses fastReruns, the other doesn't), we should make sure it works in both cases

Reply:
Yes!